tables/SV-SE-G1.utb

```diff
@@ -0,0 +1,506 @@
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+
+# Characters not to backtranslate (must be included for braille input to work)
+include braille-patterns.cti
+
+# Spaces
+space \x00a0 0
+include spaces.uti
+
+# Punctuation
+punctuation ! 235			exclamation mark		x0021
+punctuation " 56			double quote				x0022
+sign ' 5				apostrophe					x0027
+punctuation ( 236			left parenthesis		x0028
+punctuation ) 356			right parenthesis		x0029
+punctuation , 2				coma								x002C
+hyphen - 36			hyphen-minus				x002D
+punctuation . 3				point								x002E
+punctuation : 25			colon								x003A
+punctuation ; 23			semicolon						x003B
+punctuation ? 26			question mark				x003F
+punctuation [ 12356	left square bracket		x005B
+punctuation ] 23456	right square bracket	x005D
+punctuation { 6-12356	left curly bracket		x007B
+punctuation } 6-23456	right curly bracket		x007D
+punctuation « 45-2356	left pointing double angle								x00AB
+punctuation » 2356-12	right pointing double angle	x00BB
+noback punctuation	\x2010 46		 # 8208			hyphen
+noback punctuation	\x2011 46		 # 8209			non-breaking hyphen
+punctuation	\x2013 36-36		 # 8211			smart minus sign
+punctuation	\x2018 5		 # 8216			smart single left quotation mark
+sign \x2019 5		 # 8217			smart single right quotation mark
+punctuation	\x201C 56		# 8220			smart opening double quote
+punctuation	\x201D 56		# 8221			smart closing double quote
+punctuation	\x201E 56		# 8222			smart double low quotation mark
+punctuation	\x201F 56		# 8223			smart double high reverse quotation mark
+punctuation \x2026 3-3-3 # 8230			smart ellipsis
+
+# Special signs
+sign # 45-3456				number sign					x0023
+sign € 45-15
+sign $ 45-234					dollar							x0024
+sign % 1456					percent							x0025
+sign ‰ 1456-1456	promille
+sign & 12346					ampersand						x0026
+sign * 35							asterisk						x002A
+noback sign \x2606 35	white star
+sign _ 45-36						underline							x005F
+sign @ 45-12356				commercial at				x0040
+sign ^ 4						circumflex accent			x005E
+sign \\ 45-34				reverse solidus				x005C
+sign | 456					vertical line					x007C
+sign ¢ 45-14					cent sign							x00A2
+sign £ 45-123				pound sign						x00A3
+sign ¤ 45-15				currency sign					x00A4
+sign ¥ 45-13456			yen										x00A5
+sign ○ 12456-1245	degrees
+sign ∙	12456-3	multiplication
+sign § 346					paragraph							x00A7
+sign © 236-14-356		copyright		x00A9
+sign ® 236-1235-356	registered trademark
+sign ™ 236-2345-134-356	trademark
+sign ° 4-356					degree sign									x00B0
+sign ² 4-6-126				superscript 2								x00B2
+sign ³ 4-6-146				superscript 3								x00B3
+sign ¹ 4-6-16					superscript 1								x00B9
+sign \x20AC 15-136-1235-135					Euro sign

```

Comment:
For uniformity, I think it's better to replace escape sequences like \x20AC with the Unicode characters like €.